assets/trino/functions.sdf.yml

```diff
@@ -3647,6 +4421,7 @@ function:
   optional-parameters: []
   returns:
     datatype: timestamp
+  cross-link: https://prestodb.io/docs/current/functions.html
```

Comment:
@wizardxz , this `objectid_timestamp` (and probably `objectid` above it) seem to be specific to the MongoDB connector.  Should we drop them from the list for this reason? 
See https://trino.io/docs/current/connector/mongodb.html#objectid_timestamp